### ISSUE TITLE
raw link like <https://www.google.com> considered as html tag

### ISSUE THREAD

**Issue by K4zuki** [shd101wyy/markdown-preview-enhanced#577]
# Source
```markdown
<https://www.google.com>
```

# Expected
<https://www.google.com>

# Preview result
```
HTML error. Tag  not closed.  is required.
```
---
This happens at version `0.14.7`. is this intended?

**Comment by shd101wyy** [comment]
It's a big. Will be fixed in next release. Thanks.
And may I ask why you write the link inside <>?

**Comment by K4zuki** [comment]
This is from very original `markdown.pl` behavior.
GFM also uses this syntax

**Comment by shd101wyy** [comment]
Ok I will let you know once I fix it.

**Comment by K4zuki** [comment]
Thanks!

**Comment by shd101wyy** [comment]
@K4zuki Fixed.

**Comment by K4zuki** [comment]
Fix is confirmed at `0.14.8`
Thank you for your help!

Closing thread
